### PR TITLE
HDDS-11728. Refactor subcommand layouts of ozone debug and repair

### PR DESCRIPTION
**Pull request by sarvekshayr**
## What changes were proposed in this pull request?
Refactored the layout of `ozone debug` and `ozone repair` subcommands to improve organisation and usability. The new structure groups related commands logically, making it easier to find existing tools and add new ones.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-11728

## How was this patch tested?
Verified the structure and tested the patch manually.

### CODE REVIEW

**Comment by sarvekshayr** [comment]
Please review this patch - @errose28

**Comment by errose28** [comment]
Thanks for working on this @sarvekshayr. Can you share the new layout of the commands in plain text format? This will be easier for others to review before checking the code directly.

**Comment by sarvekshayr** [comment]
New layout

```
debug
│
├── OzoneDebug
│
├── chunk
│   ├── ChunkDataNodeDetails
│   ├── ChunkDetails
│   ├── ChunkKeyHandler
│   ├── ChunkType
│   └── ContainerChunkInfo
│
├── container
│   ├── ContainerCommands
│   ├── ExportSubcommand
│   ├── InfoSubcommand
│   ├── InspectSubcommand
│   └── ListSubcommand
│
├── ldb
│   ├── DBScanner
│   ├── DropTable
│   ├── ListTables
│   ├── RDBParser
│   └── ValueSchema
│
├── segmentparser
│   ├── GenericRatisLogParser
│   ├── OMRatisLogParser
│   ├── BaseLogParser
│   ├── DatanodeRatisLogParser
│   ├── RatisLogParser
│   └── SCMRatisLogParser
│
├── CompactionLogDagPrinter
├── FindMissingPadding
├── LeaseRecoverer
├── PrefixParser
├── ReadReplicas
└── VersionDebug
```
```
repair
│
├── OzoneRepair
│
├── ldb
│   ├── RDBRepair
│   ├── SnapshotRepair
│   └── TransactionInfoRepair
│
├── om
│
├── quota
│   ├── QuotaRepair
│   ├── QuotaStatus
│   └── QuotaTrigger
│
└── RecoverSCMCertificate

```

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/chunk/package-info.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * <p>
+ *
+ */
```
Comment (adoroszlai) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/chunk/package-info.java:19]: nit: license header is not javadoc, please copy from:

https://github.com/apache/ozone/blob/8e27ddec6b64d18205115a048675c707319e2df6/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/ldb/package-info.java#L1-L17

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ldb/package-info.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * <p>
+ *
+ */
```
Comment (adoroszlai) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ldb/package-info.java:19]: same nit: license header is not javadoc, please copy from:

https://github.com/apache/ozone/blob/8e27ddec6b64d18205115a048675c707319e2df6/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/ldb/package-info.java#L1-L17

**Review by adoroszlai — COMMENTED** [review]
Thanks @sarvekshayr for the patch.

**Comment by adoroszlai** [comment]
Thanks @sarvekshayr for updating the patch, LGTM.  Let's wait for other reviews (@errose28, @sumitagrawl).

**Comment by adoroszlai** [comment]
Thanks @sarvekshayr for the patch.  Any post-commit comments can be addressed in follow-up.

**Pull request merged**